extension-compose/src/main/java/com/mapbox/maps/extension/compose/MapboxMap.kt

```diff
@@ -10,6 +9,52 @@ import com.mapbox.maps.MapInitOptions
 import com.mapbox.maps.MapView
 import com.mapbox.maps.MapboxExperimental
 import com.mapbox.maps.MapboxMap
+import com.mapbox.maps.extension.compose.internal.MapApplier
+import com.mapbox.maps.extension.compose.internal.MapNode
+import com.mapbox.maps.extension.compose.internal.MapPreviewPlaceHolder
+import com.mapbox.maps.plugin.gestures.OnMapClickListener
+import com.mapbox.maps.plugin.gestures.OnMapLongClickListener
+import com.mapbox.maps.plugin.gestures.gestures
+import kotlinx.coroutines.awaitCancellation
+
+/**
+ * MapboxMapNode to observe/update the input arguments of MapboxMap.
+ */
+private class MapboxMapNode(
+  val controller: MapView,
```

Comment:
```suggestion
  internal val controller: MapView,
```

Reply:
The MapboxMapNode is a private class, so I think it's not necessary to add internal modifier to its properties.

Reply:
it's private for now. maybe in the future you will change it. so you already have proper modifiers
but it's up to you

Reply:
Imo composable nodes are internal details and shouldn't be public anyway, I've also moved this definition to internal package.